### PR TITLE
Add CRV mining

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

File: contracts/truefi/CurvePool.sol [hunk 1]
```diff
@@ -41,13 +44,24 @@ contract CurvePool is ITruePool, ERC20, ReentrancyGuard, Ownable {
...
+
     function poolValue() public view returns (uint256) {
         return
             _currencyToken.balanceOf(address(this)).add(_lender.value()).add(
-                _curvePool.token().balanceOf(address(this)).mul(_curvePool.curve().get_virtual_price()).div(1 ether)
+                totalLiquidityTokenBalance().mul(_curvePool.curve().get_virtual_price()).div(1 ether)
             );
     }
 
+    function prepareTokens(uint256 neededAmount) internal {
```
Comment (vanruch) [inline contracts/truefi/CurvePool.sol:58]: I'd rename it. But don't know how yet

Comment (vanruch) [reply contracts/truefi/CurvePool.sol]: Smth like `restock`, `refuel` maybe?

Comment (barrutko) [reply contracts/truefi/CurvePool.sol]: Renamed

Comment (barrutko) [reply contracts/truefi/CurvePool.sol]: Oh. Hmmm... I was thinking about some longer name as this is not an external method

**Review by vanruch — APPROVED** [review]

**Pull request merged**